api/Element.json

```diff
@@ -4529,6 +4578,55 @@
           }
         }
       },
+      "MozMousePixelScroll_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/MozMousePixelScroll_event",
+          "description": "<code>MozMousePixelScroll</code> event",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": true
```

Comment:
This should be false.

Reply:
This should definitely be fixed before this PR is merged.